gen/dcos-config.yaml

```diff
@@ -315,6 +315,11 @@ package:
       MESOS_GC_DELAY={{ gc_delay }}
       MESOS_HOSTNAME_LOOKUP=false
       GLOG_drop_log_memory=false
+  - path: /etc/dcos-log-config.json
+    content: |
+      {
+        "verbose": true
```

Comment:
if this is always true why does it need to be config? Why can't it just be the program's behavior/